tests/draft4/uniqueItems.json

```diff
@@ -29,6 +29,11 @@
                 "valid": false
             },
             {
+                "description": "property order of array of objects is ignored",
+                "data": [{"foo": "bar", "bar": "foo"}, {"bar": "foo", "foo": "bar"}],
+                "valid": false
+            },
```

Comment:
This (and the two array tests below) are my interpretation of the test T1 in [the paper](http://cswr.github.io/JsonSchema/spec/why), which test that schemas ignore the ordering of properties in JSON objects (according to the JSON rfc, objcet properties are not ordered) but respect the ordering of arrays (which are ordered - this wasn't in the paper but it seems important and closely related)